### DISCUSSION TITLE
Force fail on certain condition

### DISCUSSION THREAD

**Discussion by brobee** [cypress-io/cypress#18349] [Questions and Help]
Hello, I don't see any option in cypress to fail a test. I don't want to use any helper or fake a test with contains. just give me an error with custom message.

How can I do it? Thanks!

**Comment by san-slysz** [comment]
**Force to fail**
If you think 'I put if (blabla) then 'force to fail' then you are recoding what an assertion does. So just assert. subject.should(condition). This will fail. It's not that Cyrpess doesn't have it, its that its 101 on a test framework, and that's assertions job.
**Custom error message**
Assertions allows for custom error message. Just add another argument, being you error message (string). This works for should() and for expected.

If you have trouble writing the right assertion in for your situation, post a message explaining what you are trying to do precisely, and the current code you have. Else we can't help you.
